lisp/forge-commands.el

```diff
@@ -67,7 +67,12 @@ for a repository using the command `forge-add-pullreq-refspec'."
     ("c f" "fork or remote" forge-fork)]]
   [["Configure"
     ("a" "add repository to database" forge-add-repository)
-    ("r" "forge.repository" forge-forge.remote)]])
+    ("r" "forge.repository" forge-forge.remote)
+    ("s" (lambda ()
```

Comment:
I would like to avoid a lambda here. If you define `forge-toggle-display-sections` using `define-transient-suffix`, then you could move this here. You could also define this as a named function, but that might be a bit much.